arch/arm64/core/Kconfig

```diff
@@ -158,6 +158,21 @@ config ARMV8_A
 	  so that it can support some features included in the AArch64 state.
 	  It supports the T32 and A32 instruction sets.
 
+config XEN
+	bool
+	default y
+	depends on DT_HAS_XEN_XEN_ENABLED
+	help
+	  Enables support of Xen hypervisor on arm64 platform. Get enabled
+	  when board device tree contains "hypervisor" node with "xen,xen"
+	  compatible enabled.
```

Comment:
Oh man, this is super weird to have here.

Ok, it's time to have XEN as first class citizen then. Can you move all the XEN related stuff into a `xen/Kconfig.xen` file that you `rsource` from here? At least we do not pollute the general `Kconfig` too much.

Reply:
@carlocaione had such ideas too during refactor. I will fix it soon along with CI. Thank you.